index.js

```diff
@@ -19,72 +22,79 @@ const maxNumberOfAttempts = 5;
-  }
-
-  if (guess !== targetNumber) {
-    if (guess < targetNumber) {
-      tooLowMessage.style.display = '';
-    } else {
-      tooLowMessage.style.display = '';
+    // Get value from guess input element
+    const guess = parseInt(guessInput.value, 10);
+    attempts = attempts + 1;
+    guessInput.value = '';
+    resetButton.style.display = '';
+
+    hideAllMessages();
+
+    if (guess === targetNumber) {
+        numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
+        numberOfGuessesMessage.style.display = '';
+        correctMessage.style.display = '';
+        submitButton.disabled = true;
+        guessInput.disabled = true;
     }
 
-    const remainingAttempts = maxNumberOfAttempts - attempts;
-
-    numberOfGuessesMessage.style.display = '';
-    numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guesses remaining`;
-  }
-
-  if (attempts ==== maxNumberOfAttempts) {
-    submitButton.disabled = true;
-    guessInput.disabled = true;
-  }
-
-  guessInput.value = '';
+    if (guess !== targetNumber) {
+        if (guess < targetNumber) {
+            tooLowMessage.style.display = '';
+        } else {
+            tooHighMessage.style.display = '';
+        }
+        let remainingAttempts = maxNumberOfAttempts - attempts;
+        numberOfGuessesMessage.style.display = '';
+        let remainingAttemptsMessage = remainingAttempts === 1 ? `${remainingAttempts} guess` : `${remainingAttempts} guesses`;
+        numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttemptsMessage} remaining`;
+    }
 
-  resetButton.style.display = '';
+    if (attempts === maxNumberOfAttempts) {
+        maxGuessesMessage.style.display = '';
+        submitButton.disabled = true;
+        guessInput.disabled = true;
+    }
 }
 
 function hideAllMessages() {
-  for (let elementIndex = 0; elementIndex <= messages.length; elementIndex++) {
-    messages[elementIndex].style.display = 'none';
-  }
+    for (let elementIndex = 0; elementIndex < messages.length; elementIndex++) {
+        messages[elementIndex].style.display = 'none';
+    }
 }
 
-funtion setup() {
-  // Get random number
-  targetNumber = getRandomNumber(1, 100);
-  console.log(`target number: ${targetNumber}`);
+function setup() {
+    // Get random number
+    targetNumber = getRandomNumber(1, 100);
+    console.log(`target number: ${targetNumber}`);
 
-  // Reset number of attempts
-  maxNumberOfAttempts = 0;
+    // Reset number of attempts
+    attempts = 0;
 
-  // Enable the input and submit button
-  submitButton.disabeld = false;
-  guessInput.disabled = false;
+    // Enable the input and submit button
+    submitButton.disabled = false;
+    guessInput.disabled = false;
 
-  hideAllMessages();
-  resetButton.style.display = 'none';
+    hideAllMessages();
+    resetButton.style.display = 'none';
 }
 
-submitButton.addEventListener('click', checkGuess);
+guessInput.addEventListener('change', e => {
```

Comment:
I need to click to clear the disabled submit button, just changing the number doesn't work.  It also accepts floating point numbers. (even just the letter 'e' which says, you guessed NaN).  So check for integer and maybe a different event would catch the edit of the number.